.ado/pipelines/templates/jobs-workload-deploy.yaml

```diff
@@ -114,6 +114,10 @@ jobs: # using jobs so they each run in parallel
             $fullContainerImageName = Get-Content -Path "$(Pipeline.Workspace)/catalogservice-containerImageName/catalogservice.txt"
             echo "*** Retrieved full container image name from artifact - $fullContainerImageName"
 
+            # Private Endpoints IP Address Range - used to limit egress traffic in the K8s cluster
+            # $privateEndpointsIPAddressRange = $stamp.privatendpoints_addressrange
+            # echo "*** Egress traffic will be restricted to $privateEndpointsIPAddressRange (using Network Policy)"
```

Comment:
so should this stay in here as examples or can it go out?

Reply:
if it stays in, please add a comment explaining

Reply:
modified the existing comment a bit